checks/system/unix.py

```diff
@@ -221,6 +221,31 @@ def check(self, agentConfig):
             return False
 
 
+class File_Handlers(Check):
```

Comment:
nit: could you name this class `FileHandles` instead? Just to be consistent with the standard case of python classes, and also name it `handles` instead of `handlers`